Rocket.Chat/Models/LoginService.swift

```diff
@@ -43,6 +47,14 @@ class LoginService: BaseModel {
     @objc dynamic var loginStyle: String?
     @objc dynamic var buttonColor: String?
 
+    // CAS
+
+    @objc dynamic var loginUrl: String?
+
+    var isValid: Bool {
+        return !(serverUrl?.isEmpty ?? true) || type == .saml || type == .cas
```

Comment:
I think this method deserves a comment... it validates if the connected server has any valid authentication method?

Reply:
It validates whether the LoginService has enough information to be used. I'll add a comment.